subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -10,28 +10,396 @@
 #define LOG_MODULE_NAME bt_ctlr_ull_adv_iso
 #include "common/log.h"
 #include "hal/debug.h"
+#include "hal/cpu.h"
+#include "hal/ccm.h"
+#include "hal/ticker.h"
+
+#include "util/util.h"
+#include "util/memq.h"
+#include "util/mayfly.h"
+#include "util/mem.h"
+#include "util/mfifo.h"
+#include "ticker/ticker.h"
+
+#include "pdu.h"
+#include "ll.h"
+#include "lll.h"
+
+#include "lll_vendor.h"
+#include "lll_adv.h"
+#include "lll_conn.h"
+
+#include "ull_internal.h"
+#include "ull_adv_types.h"
+#include "ull_adv_internal.h"
+
+static struct ll_adv_iso ll_adv_iso[CONFIG_BT_CTLR_ADV_SET];
+static void *adv_iso_free;
+
+static MFIFO_DEFINE(iso_tx, sizeof(struct lll_tx),
+		    CONFIG_BT_CTLR_ISO_TX_BUFFERS);
+
+/* TODO: Share between BIS and CIS */
+static struct {
+	void *free;
+	uint8_t pool[CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE *
+			CONFIG_BT_CTLR_ISO_TX_BUFFERS];
+} mem_iso_tx;
```

Comment:
This should really in a file that is common for BIS and CIS (cf. #30000, which creates such a file).

Reply:
#30000 is merged now.

Reply:
Added a commit in this PR that moved it to `ull_iso.c`